### PR TITLE
Optimizer: simplify unary positive sign

### PR DESCRIPTION
**Pull request by GeorgeSittas**

### CODE REVIEW

File: sqlglot/optimizer/simplify.py [hunk 1]
```diff
@@ -239,6 +239,10 @@ def simplify_literals(expression):
             if value[0] == "-":
                 return exp.Literal.number(value[1:])
             return exp.Literal.number(f"-{value}")
+    elif isinstance(expression, exp.Pos):
+        this = expression.this
+        if this.is_number:
```
Comment (tobymao) [inline sqlglot/optimizer/simplify.py:244]: what about the case 

+ - 1 ?

Comment (tobymao) [reply sqlglot/optimizer/simplify.py:244]: ```+-1```

Comment (GeorgeSittas) [reply sqlglot/optimizer/simplify.py:244]: Added some more tests for these.

**Review by barakalon — APPROVED** [review]

File: tests/fixtures/optimizer/simplify.sql [hunk 2]
```diff
@@ -274,6 +274,15 @@ TRUE;
 -(-1);
 1;
 
+- -+1;
+1;
+
++-1;
```
Comment (tobymao) [inline tests/fixtures/optimizer/simplify.sql:280]: is it correct? it doesn’t negate it? is + a noop?

File: sqlglot/generator.py [hunk 3]
```diff
@@ -1128,7 +1128,7 @@ def neg_sql(self, expression):
         return f"-{sep}{this_sql}"
 
     def pos_sql(self, expression):
-        return f"+{self.sql(expression, 'this')}"
+        return f"{self.sql(expression, 'this')}"
```
Comment (tobymao) [inline sqlglot/generator.py:1131]: you don't need the f

**Review by tobymao — APPROVED** [review]

**Pull request merged**